doc/nrf/installation/recommended_versions.rst

```diff
@@ -18,79 +18,41 @@ Supported operating systems
 ***************************
 
 The |NCS| supports Microsoft Windows, Linux, and macOS for development.
-The following table shows the operating system versions that support the |NCS| tools:
-
-.. os_table_start
-
-.. list-table::
-   :header-rows: 1
-
-   * - Operating System
-     - x86
-     - x64
-     - ARM64
-   * - `Windows 11`_
-     - Tier 3
-     - Tier 3
-     - Not supported
-   * - `Windows 10`_
-     - Tier 3
-     - Tier 1
-     - Not supported
-   * - `Linux - Ubuntu 22.04 LTS`_
-     - Not supported
-     - Tier 1
-     - Not supported
-   * - `Linux - Ubuntu 20.04 LTS`_
-     - Not supported
-     - Tier 2
-     - Not supported
-   * - `macOS 14`_
-     - Not applicable
-     - Tier 3
-     - Tier 3
-   * - `macOS 13`_
-     - Not applicable
-     - Tier 1
-     - Tier 1
-   * - `macOS 12`_
-     - Not applicable
-     - Tier 3
-     - Tier 3
-   * - `macOS 11`_
-     - Not applicable
-     - Tier 2
-     - Tier 2
-   * - `macOS 10.15`_
-     - Not applicable
-     - Tier 3
-     - Not supported
-
-.. os_table_end
+The following tables list the support levels for tools and firmware.
 
 Tier definitions
-  The table uses several tier definitions to categorize the level of operating system support:
+  The tables use the following tier definitions to categorize the level of operating system support:
 
   .. toggle:: Support levels
 
-     Tier 1
-       The |NCS| tools will always work.
-       The automated build and automated testing ensure that the |NCS| tools build and successfully complete tests after each change.
-
-     Tier 2
-       The |NCS| tools will always build.
-       The automated build ensures that the |NCS| tools build successfully after each change.
-       There is no guarantee that a build will work because the automation tests do not always run.
-
-     Tier 3
-       The |NCS| tools are supported by design, but are not built or tested after each change.
-       Therefore, the application may or may not work.
-
-     Not supported
-       The |NCS| tools do not work, but it may be supported in the future.
-
-     Not applicable
-       The specified architecture is not supported for the respective operating system.
+     .. list-table::
+        :header-rows: 1
+
+        * - Tier level
+          - Building
+          - Testing
+          - Failures
+        * - **Tier 1 (recommended)**
+          - Successfully executed based on risk.
+          - Successfully executed based on risk.
+          - Any failures block the release until they are fixed or a viable workaround is found.
+        * - **Tier 2**
+          - Building on this version is successfully executed at the :ref:`minimum recommended version of tools <requirements_toolchain>`.
+          - Manual Sanity Check is done based on risk.
```

Comment:
```suggestion
          - A manual sanity check is done based on risk.
```

Reply:
I got the capitalized wording from @shanthanordic as approved by both her and @ihansse , so I'm not making this change.

@shanthanordic , do we need to add a definition of what Manual Sanity Check is to justify the capitalization?